README.md

```diff
@@ -37,6 +37,68 @@ To cross-compile for iOS, clone [ios-cmake](https://github.com/leetal/ios-cmake)
 | i386         | `scripts/build-ios-i386.sh`   | iPhone Simulator (32-bit) |
 | x86_64       | `scripts/build-ios-x86_64.sh` | iPhone Simulator (64-bit) |
 
+## End-to-End Benchmarking
+
+Caffe2 backend of PyTorch 1.0 natively integrates QNNPACK, and provides a [pre-trained quantized MobileNet v2 model](https://github.com/caffe2/models/tree/master/mobilenet_v2_quantized). Below are instructions for benchmarking this model end-to-end with QNNPACK.
+
+### ARMv7 (32-bit) Android
+
+```bash
+# Clone PyTorch 1.0 repo
+git clone --recursive https://github.com/pytorch/pytorch.git
+cd pytorch
+
+# Optional: update QNNPACK submodule to latest revision
+git submodule update --remote third_party/QNNPACK
+
+# Build Caffe2 (including binaries) for Android
+scripts/build_android.sh -DANDROID_TOOLCHAIN=clang -DBUILD_BINARY=ON
```

Comment:
btw, I have been using confu + ninja, what is our plan on it?

Reply:
this builds Caffe2, not QNNPACK. QNNPACK has build scripts for both confu+ninja and CMake.